types/node-notifier/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for node-notifier 5.4.0
+// Type definitions for node-notifier 5.4.1
```

Comment:
This should be only major and minor version (one of the lint rules catches this but currently they're all disabled in the tslint.json file in this folder). So you can either leave it as it is or change it to 5.4 or 6.0 (the latest package version).

Reply:
Could you fix it? I'm away from the computer for a few days